static/prelogin/less/components/pricing-section.less

```diff
@@ -170,7 +170,7 @@ row heights can be found in components/pricing.less
 
   @media (max-width: @smallest-table-width) {
     overflow-x: scroll;
-    overflow-y: none;
+    overflow-y: hidden;
```

Comment:
Great - two last small tweaks (sorry for not making this clearer in the
spec):

- remove "Priority support" from the short view  of Advanced
- remove "Dedicated Account Manager" from Enterprise

thanks!

On Fri, Apr 29, 2016 at 9:22 AM, Jenny Schweers <notifications@github.com>
wrote:

> In static/prelogin/less/components/pricing-section.less
> <https://github.com/dimagi/commcarehq-prelogin/pull/74#discussion_r61576686>
> :
>
> > @@ -170,7 +170,7 @@ row heights can be found in components/pricing.less
> >
> >    @media (max-width: @smallest-table-width) {
> >      overflow-x: scroll;
> > -    overflow-y: none;
> > +    overflow-y: hidden;
>
> I don't know, but @biyeun <https://github.com/biyeun> might
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/dimagi/commcarehq-prelogin/pull/74/files/a4871d419b33de507f636bf76d6de61eaa744555#r61576686>
>